code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -144,19 +204,47 @@ func (sn *BlobberChallenge) addChallenge(challenge *StorageChallenge) bool {
 		sn.ChallengeMap[challenge.ID] = challenge
 		return true
 	}
+
 	return false
 }
 
 type StorageChallenge struct {
-	Created        common.Timestamp   `json:"created"`
-	ID             string             `json:"id"`
-	PrevID         string             `json:"prev_id"`
-	Validators     []*ValidationNode  `json:"validators"`
-	RandomNumber   int64              `json:"seed"`
-	AllocationID   string             `json:"allocation_id"`
-	Blobber        *StorageNode       `json:"blobber"`
+	Created common.Timestamp `json:"created"` // needed
```

Comment:
Why do we need anything other than ID and Created here *(for `canceledPassRates`)? 

This needs to be as slim as possible or ideally removed altogether. Most of this data is to be stored in the event database not the MPT. Only keep what is absolutely essential in the MPT.